src/overmap.cpp

```diff
@@ -160,8 +160,6 @@ void load_overmap_specials(JsonObject &jo)
     }
 
     spec.rotatable = jo.get_bool("rotate", false);
-    spec.unique = jo.get_bool("unique", false);
-    spec.required = jo.get_bool("required", false);
```

Comment:
Those two members of `overmap_special` are now uninitialized, but luckily nothing is reading them. They should probably be removed.